src/middlewares/auth.ts

```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response} from 'express';
+import jwt from 'jsonwebtoken';
+import { verifyToken } from '../utils/jwt.handle';
+
+export default function validateToken(req: Request, res: Response, next: NextFunction) {
+  try {
+    let token = req.headers.authorization || '';
+    if (token.startsWith('Bearer ')) {
+      token = token.slice(7, token.length);
+    }
+    const payload = verifyToken(token) as { id: string };
```

Comment:
seria mas adecuado ponerle el tipo id number?
```suggestion
    const payload = verifyToken(token) as { id: number };
```

Reply:
realizado